convert2rhel/subscription.py

```diff
@@ -833,15 +833,15 @@ def _relevant_subscription_manager_pkgs():
             "python3-syspurpose",
             "python3-cloud-what",
             "json-c.x86_64",  # there's also an i686 version we don't need unless the json-c.i686 is already installed
-            "subscription-manager-rhsm-certificates.noarch",
+            "subscription-manager-rhsm-certificates",
         ]
 
     elif system_info.version.major >= 9:
         relevant_pkgs += [
             "libdnf-plugin-subscription-manager",
             "python3-subscription-manager-rhsm",
             "python3-cloud-what",
-            "subscription-manager-rhsm-certificates.noarch",
```

Comment:
Given we had `.x86_64` before in #991 we're not going to encounter any issues?

Reply:
We won't encounter issues anymore on EL9 because https://issues.redhat.com/browse/RHELDST-21855 fixed both EL8 and EL9 UBI repos. I should have stated that in the PR desc 🫣